### PR TITLE
[4.0] Taking off CPanel Toolbar Icons from language strings for core Dashboards

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/26217

### Summary of Changes
Hard coding icons class values for CPanel Icons and deleting the afferent language strings.
It does not destroy the capacity for 3pd to use the feature for custom dashboards with lang strings.

Adding `COM_CPANEL_DASHBOARD_PRIVACY_TITLE="Privacy Dashboard"` as it was missing from ini file.

### Testing Instructions
Display various core CPanel dashboards and verify that the Toolbar Icons are present.

PLT: please take a decision on this as adding classes values in language strings, at least for core, is not a good idea.

@wilsonge @HLeithner

### CODE REVIEW

**Comment by brianteeman** [comment]
Can you add an icon for the Help page - beforw this PR it used incorrectly the home icon. after this PR it is empty

otherwise all good

**Comment by Hackwar** [comment]
As I explained in the chat, I'm against adding this. This is yet another special case for core, which third party developers have no access to and in the past we had a fantastic track record of breaking stuff that we didn't have in core. Don't introduce special treatment for the core components.

**Comment by brianteeman** [comment]
> Don't introduce special treatment for the core components.

agreed

BUT putting css as a language string is worse - translators have a great track record of translating everything

**Comment by infograf768** [comment]
help icon added
Screen Shot 2019-09-10 at 12 35 24

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on b854c88eb1470057477bc8f2ce92d74ab289e212<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26252">issues.joomla.org/tracker/joomla-cms/26252</a>.</sub>

File: administrator/components/com_cpanel/View/Cpanel/HtmlView.php [hunk 1]
```diff
@@ -95,10 +95,53 @@ public function display($tpl = null)
...
+				}
+				else
+				{
+					$icon = '';
+				}
+			}
+			else
+			{
+				// Search for Icon
+				if ($lang->hasKey($keyIcon))
```
Comment (Quy) [inline administrator/components/com_cpanel/View/Cpanel/HtmlView.php]: Suggestion: Move line 141 to 138 `elseif ($lang->hasKey($keyIcon))` to reduce one level indentation.

Comment (infograf768) [reply administrator/components/com_cpanel/View/Cpanel/HtmlView.php]: will do.

Comment (infograf768) [reply administrator/components/com_cpanel/View/Cpanel/HtmlView.php]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c85bf2fa91d8c1e16e746b694af951922ce31b9f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26252">issues.joomla.org/tracker/joomla-cms/26252</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26252">issues.joomla.org/tracker/joomla-cms/26252</a>.</sub>

**Comment by shraddhaSankpal27** [comment]
I have tested this item :white_check_mark: successfully on bf83031636ec8f81f811e8ad2da77dde32958892<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26252">issues.joomla.org/tracker/joomla-cms/26252</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**